ocs_ci/ocs/couchbase.py

```diff
@@ -165,42 +139,89 @@ def test_couchbase_workload_simple(self, pillowfight):
             3,
             get_pod_name_by_pattern,
             'couchbase-operator',
-            self.COUCHBASE_OPERATOR
+            constants.COUCHBASE_OPERATOR
         ):
             try:
                 if self.is_up_and_running(couchbase_pod[0], self.up_check):
                     break
             except IndexError:
                 log.info("Couchbase operator is not up")
+
         cb_work = templating.load_yaml(constants.COUCHBASE_WORKER_SECRET)
         self.cb_worker = OCS(**cb_work)
         self.cb_worker.create()
+
+    def create_couchbase_worker(self, replicas=1):
+        """
+        Deploy a Couchbase server and pillowfight workload using operator
+
+        The couchbase workers do not come up unless there is an admission controller
+        running.  The admission controller is started from the default project prior
+        to bringing up the operator.  Secrets, rolebindings and serviceaccounts
+        need to also be generated.
+
+        Once the couchbase operator is running, we need to wait for the three
+        worker pods to also be up.  Then a pillowfight task is started.
+
+        After the pillowfight task has finished, the log is collected and
+        analyzed.
+
+        Raises:
+            Exception: If pillowfight results indicate that a minimum performance
+                level is not reached (1 second response time, less than 1000 ops
+                per second)
+
+        """
+        logging.info('Running pre reqs')
+        self._prereq()
+        logging.info('Creating pods..')
         cb_example = templating.load_yaml(constants.COUCHBASE_WORKER_EXAMPLE)
+        cb_example['spec']['servers'][0]['size'] = replicas
         self.cb_examples = OCS(**cb_example)
         self.cb_examples.create()
+
         # Wait for last of three workers to be running.
-        for cb_wrk_pod in TimeoutSampler(
+
+        logging.info('Waiting for the pods to Running')
+        for cb_wrk_pods in TimeoutSampler(
             self.WAIT_FOR_TIME,
             3,
```

Comment:
Same comment.
Is this replica count? or expected pods to spin up?

Reply:
Nope, sleep time ,https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L734